### PR TITLE
[GUI] Added new notification center with popups

### LINKED ISSUE #1098: Notification center in the new design

### PR DESCRIPTION
**Pull request by Max-7**
Closes #1098

### CODE REVIEW

File: misc/format_material.py [hunk 1]
```diff
@@ -20,6 +20,10 @@ def resize_imgs(input_dir):
         root = tree.getroot()
         root.attrib["width"] = "144"
         root.attrib["height"] = "144"
+        for child in root:
+            if child.attrib.get("fill", None) and child.attrib.get("fill", "") != "#000000":
+                print("Yep")
```
Comment (touilleMan) [inline misc/format_material.py:25]: nope ?

File: misc/format_material.py [hunk 2]
```diff
@@ -20,6 +20,10 @@ def resize_imgs(input_dir):
         root = tree.getroot()
         root.attrib["width"] = "144"
         root.attrib["height"] = "144"
+        for child in root:
```
Comment (touilleMan) [inline misc/format_material.py:23]: I guess this is done to enforce transparency, but we should add a comment to explain this

File: parsec/core/gui/tr/parsec_fr.po [hunk 3]
```diff
@@ -1765,7 +1765,7 @@ msgstr "Fermer"
 msgid "TEXT_NEW_VERSION_UP_TO_DATE"
 msgstr "Votre application est à jour."
 
-msgid "LABEL_NOTIFICATIONS"
+msgid "TEXT_NOTIFICATIONS_TITLE"
```
Comment (touilleMan) [inline parsec/core/gui/tr/parsec_fr.po:1768]: Translation is missing

File: parsec/core/gui/tr/parsec_en.po [hunk 4]
```diff
@@ -1758,7 +1758,7 @@ msgstr "Close"
 msgid "TEXT_NEW_VERSION_UP_TO_DATE"
 msgstr "Your PARSEC is up-to-date!"
 
-msgid "LABEL_NOTIFICATIONS"
+msgid "TEXT_NOTIFICATIONS_TITLE"
```
Comment (touilleMan) [inline parsec/core/gui/tr/parsec_en.po:1761]: Text is missing:
image

File: parsec/core/gui/tr/parsec_en.po [hunk 5]
```diff
@@ -1335,34 +1335,34 @@ msgstr "Register a device"
...
+msgstr "Your role has changed on the workspace {workspace}."
 
-msgid "NOTIF_INFO_WORKSPACE_UNSHARED_{}"
-msgstr ""
+msgid "TEXT_NOTIF_INFO_WORKSPACE_UNSHARED_workspace"
+msgstr "You no longer have access to {workspace}."
 
-msgid "NOTIF_WARN_SYNC_CONFLICT_{}"
+msgid "TEXT_NOTIF_WARN_SYNC_CONFLICT_path"
 msgstr ""
```
Comment (touilleMan) [inline parsec/core/gui/tr/parsec_en.po:1366]: Text missing

File: parsec/core/gui/tr/parsec_fr.po [hunk 6]
```diff
@@ -1340,34 +1340,34 @@ msgstr "Enregistrer un appareil"
...
+msgstr "Votre rôle a changé sur l'espace de travail {workspace}."
 
-msgid "NOTIF_INFO_WORKSPACE_UNSHARED_{}"
-msgstr ""
+msgid "TEXT_NOTIF_INFO_WORKSPACE_UNSHARED_workspace"
+msgstr "Vous n'avez plus accès à l'espace de travail {workspace}."
 
-msgid "NOTIF_WARN_SYNC_CONFLICT_{}"
+msgid "TEXT_NOTIF_WARN_SYNC_CONFLICT_path"
 msgstr ""
```
Comment (touilleMan) [inline parsec/core/gui/tr/parsec_fr.po:1371]: Translation is missing

File: parsec/core/gui/central_widget.py [hunk 7]
```diff
@@ -32,7 +34,7 @@ class CentralWidget(QWidget, Ui_CentralWidget):
         "mountpoint.remote_error",
         "mountpoint.unhandled_error",
         "sharing.updated",
-        "fs.entry.file_update_conflicted",
+        "fs.entry.file_conflict_resolved",
```
Comment (touilleMan) [inline parsec/core/gui/central_widget.py:37]: I can't see where this event is triggered (didn't find any reference of `fs.entry.file_update_conflicted` either...)
Are you sure this event is still in use ? or is it for future use ?

Comment (Max-7) [reply parsec/core/gui/central_widget.py:37]: file_update_conflicted does not exist anymore, but file_conflict_resolved is a new event (https://github.com/Scille/parsec-cloud/pull/1095)

**Comment by Max-7** [comment]
Deprecated

**Pull request closed without merging**